Extensions/DevTeam.Extensions.EntityFrameworkCore.Relational/DevTeam.Extensions.EntityFrameworkCore.Relational.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageId>DevTeam.EntityFrameworkExtensions.Relational</PackageId>
```

Comment:
Please change the PackageId to DevTeam.Extensions.EntityFrameworkCore.Relational. The old name for the EntityFrameworkCore package is an obsolete and we don't want to lose statistics for it that's why we still keep it DevTeam.EntityFrameworkExtensions.

Reply:
Please also add .Extensions. (the same as the folder name)